be/src/runtime/exec_env_init.cpp

```diff
@@ -178,6 +180,18 @@ Status ExecEnv::_init_mem_tracker() {
     LOG(INFO) << "Using global memory limit: " << PrettyPrinter::print(bytes_limit, TUnit::BYTES);
     RETURN_IF_ERROR(_disk_io_mgr->init(_mem_tracker));
     RETURN_IF_ERROR(_tmp_file_mgr->init(DorisMetrics::metrics()));
+
+    int64_t storage_cache_limit = ParseUtil::parse_mem_spec(
+        config::storage_page_cache_limit, &is_percent);
+    if (storage_cache_limit > MemInfo::physical_mem()) {
+        LOG(WARNING) << "Config storage_page_cache_limit is greater than memory size, config="
+            << config::storage_page_cache_limit
+            << ", memory=" << MemInfo::physical_mem();
+    }
+    StoragePageCache::create_global_cache(storage_cache_limit);
```

Comment:
Is that OK to set cache size larger than physical memory size?

Reply:
May be OK or may be not. Now we LOG WARNING here, we should review all cache/buffer memory in our system.